tracing-modality/src/common/attr_handlers.rs

```diff
@@ -1,15 +1,81 @@
-use std::borrow::Cow;
+//! Attribute Handlers
+//!
+//! Attribute handlers allow for project-specific modifications of `tracing` data
+//! provided when events occur.
+//!
+//! This is typically needed when data needs to be modified in some way before reporting
+//! to modality, or if a translation from "project specific" keys or values to "modality
+//! specific" keys or values are necessary.
+//!
+//! Handlers are matched on a SPECIFIC key text. If a matching key text is found, the
+//! provided handler will be called, will replace the key and value of the data reported
+//! by `tracing`.
```

Comment:
What is "SPECIFIC" supposed to be drawing my attention to? It seems like it's important since it's in all caps. But I don't understand what distinction you're trying to make between that and "Handlers are matched by key."